s3store/s3store.go

```diff
@@ -536,3 +549,21 @@ func isAwsError(err error, code string) bool {
 	}
 	return false
 }
+
+func (store S3Store) CalcPartSize(size *int64) (effectivePartSize int64) {
```

Comment:
If you don't like it, we can drop it.
If there is a good reason to refrain from it, then please elaborate, as I am always eager to learn something new!